public/static/locales/en/donate.json

```diff
@@ -58,5 +58,6 @@
   "noProjectsFound": "No projects found",
   "inTimeOfTaxReturns": "in time for tax returns.",
   "titleToShare": "Planting trees against the climate crisis!",
-  "textToShare": "Preventing the climate crisis requires drastically reducing carbon emissions and planting trees. That’s why I just planted some. Check out {{linkToShare}} if you want to plant some too! #StopTalkingStartPlanting #TrillionTrees @PftP_int"
+  "textToShare": "Preventing the climate crisis requires drastically reducing carbon emissions and planting trees. That’s why I just planted some. Check out {{linkToShare}} if you want to plant some too! #StopTalkingStartPlanting #TrillionTrees @PftP_int",
+  "fortreeAmountTrees": "{{amount}}"
```

Comment:
Adding a new translation resource which contains just a variable is not necessary and just add unnecessary complexity to our code (and more work for translators).

Reply:
This is resolved